api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TransferredInCohortQueries.java

```diff
@@ -107,7 +107,7 @@ public CohortDefinition getTransferredInPatients() {
     cd.addSearch("homeVisitTrfOut", EptsReportUtils.map(homeVisitTrfOut, mappingsHomeVisitTrfOut));
     cd.addSearch("clinicalVisit", EptsReportUtils.map(clinicalVisit, mappingsClinicalVisit));
 
-    cd.setCompositionString("(transferredIn OR transferredOut) AND clinicalVisit AND NOT txCurr");
```

Comment:
This criteria should remain. You should check if the patients had a pick up during the period

Reply:
Fixed: Implemented the filter of drug_pickup